girder/models/user.py

```diff
@@ -127,7 +138,16 @@ def validate(self, doc):
 
         return doc
 
-    def authenticate(self, login, password):
+    def filter(self, doc, user, additionalKeys=None):
```

Comment:
@girder/developers I'm not sure if I like what's being done here, but it's the only way I see to created 'derived' values when a document is filtered.

Reply:
@zachmullen Let me know if you have any thoughts.